app/commands/decidim/odoo/sync_user.rb

```diff
@@ -44,7 +44,7 @@ def create_user!
       end
 
       def update_user!
-        user.nickname = Decidim::UserBaseEntity.nicknamize(odoo_info[:vat], organization: user.organization) if odoo_info[:vat].present?
+        user.nickname = Decidim::UserBaseEntity.nicknamize(odoo_info[:name], organization: user.organization) if odoo_info[:name].present?
```

Comment:
```suggestion
        user.nickname = Decidim::UserBaseEntity.nicknamize(odoo_info[:name], organization: user.decidim_organization_id)
```
Looks like needs the id specifically?

Note also that the nickname is mandatory and it should only be updated if not existing yet! so the "if" condition needs to be changed